packages/aws-rfdk/lib/lambdas/nodejs/lib/aws-lambda/types.ts

```diff
@@ -13,4 +13,5 @@
 export interface LambdaContext {
   readonly logGroupName: string;
   readonly logStreamName: string;
+  getRemainingTimeInMillis?(): number;
```

Comment:
Why optional? afaik, it shouldn't break anything to make this required -- it is actually required... we just never used it before.

Reply:
Making it not optional breaks the unit-tests of [pad-efs-storage](https://github.com/aws/aws-rfdk/blob/mainline/packages/aws-rfdk/lib/lambdas/nodejs/pad-efs-storage/test/handlers.test.ts#L129). I could add something like `getRemainingTimeInMillis: () => 1000,` to the unit-tests and remove optional. Do you agree with such a fix?

Reply:
Yeah, that sounds perfectly reasonable to me.